templates/examples/extensions/py_lambda_hooks/CanvasHook/README.md

```diff
@@ -0,0 +1,118 @@
+# Integration with Canvas LMS -- an early example implementation
+
+Students use their schools' learning management solution (LMS) to keep track of their assignments, grades, and working through their course work. To make it easier for students to stay on track and also have easy access to a knowledge base, and help with their learning progress, you can now integrate the open source AWS QnABot solution with Canvas LMS, and enable students with in-the-moment support. With this integration, students will be able to ask the chatbot about their grades, syllabus, enrollments, assignments, and announcements. 
```

Comment:
Nit: Is LMS acronym for learning management system typically or learning management solution?